plugin/trino-pulsar/pom.xml

```diff
@@ -0,0 +1,274 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>trino-root</artifactId>
+        <groupId>io.trino</groupId>
+        <version>358-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-pulsar</artifactId>
+    <description>Trino - Pulsar Connector</description>
+    <packaging>trino-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <pulsar.version>2.7.2</pulsar.version>
+        <bookkeeer.version>4.12.0</bookkeeer.version>
+        <netty.version>4.1.60.Final</netty.version>
+        <air.check.skip-dependency>true</air.check.skip-dependency>
+        <air.check.fail-checkstyle>false</air.check.fail-checkstyle>
+        <air.check.skip-duplicate-finder>true</air.check.skip-duplicate-finder>
+        <air.check.skip-modernizer>true</air.check.skip-modernizer>
```

Comment:
Please try to enable all of these airbase related properties. For example `air.check.skip-duplicate-finder` skip is dangerous as it points the issues with dependencies that could manifest on production. That would be unfortunate.

Reply:
Spent a lot of time try to solve the dependency issue, tried couple shaded version but didn't work well. (It depends on un-shade dependency and will cause NoSuchMethod at run time.)
Currently still need to skip dependency check and duplicate check, 
As Pulsar is using dep in Netty 4.1.63 and need to pin to this version, which will fail dependency check.
And some Jakarta/Javax library will cause duplicate to fail.
But I tried to run the connector with latest Trino version and seems working fine.